scripts/init.sh

```diff
@@ -141,6 +141,9 @@ echo "Migration init"
 /starhub-bin/starhub migration init
 echo "Migration migrate"
 /starhub-bin/starhub migration migrate
+
+echo "Trigger multisync once in background"
```

Comment:
should we put this command after csghub server startup?

Reply:
no， csghub server will block after run